Changed note (#101963)
Azure/azure-content: articles/azure-monitor/alerts/itsm-connector-secure-webhook-connections-azure-configuration.md

To learn more about action groups, see [Create and manage action groups in the Azure portal](../alerts/action-groups.md).

> [!NOTE]

> > Notice that when you are defining a query in Log Search alerts you need to have in the query result the Configuration items names with one of the label names "Computer",  "Resource", "_ResourceId" or "ResourceId”. This mapping will enable to map the configuration items to the ITSM payload.

Reviewer: ```suggestion
> To map the configuration items to the ITSM payload when you define a Log Search alerts query, the query result must be included in the **Configuration items**, with one of these labels:
> - "Computer"
> - "Resource"
> - "_ResourceId"
> - "ResourceId”
```
